package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "root",
```

Comment:
Is `"root"` the most appropriate name? Or should it be `"keyman"`? I am unsure!

Reply:
That's `lerna`'s package.json, basically a 'pseudo-package'. It's used to facilitate its NPM operations.

I could experiment and see if it'll function correctly should I alter it.  I figure `lerna` operations may search the file-tree to find the 'root' package, so I didn't want to take a chance with that yet.